k6/README.md

```diff
@@ -0,0 +1,14 @@
+# K6 ReadMe
+
+## Introduction
+
+This readme is for how to install and us k6 for manual load testing
+
+## Installation
+
+The link for installation of k6 can be found [here](https://grafana.com/docs/k6/latest/get-started/installation/). Follow the instructions on the link for your installer of choice
```

Comment:
Generally speaking, linking the word "here" is not good practice. The internet is full of it though 😢

Better to have the link text describe what it is linking to:

```
[Download and install k6](https://grafana.com/docs/k6/latest/get-started/installation/)
```